qt6: opt DataVisualization into consistent subdirectory QML imports

Set QTP0004 locally so designer components receive qmldir redirect files.
https://doc.qt.io/qt-6/qt-cmake-policy-qtp0004.html

--- a/qtdatavis3d/src/datavisualizationqml/CMakeLists.txt
+++ b/qtdatavis3d/src/datavisualizationqml/CMakeLists.txt
@@ -23,6 +23,8 @@
 set_source_files_properties(${qml_files} PROPERTIES
     QT_QML_SKIP_QMLDIR_ENTRY TRUE
 )
+
+qt_policy(SET QTP0004 NEW)
 
 qt_internal_add_qml_module(DataVisualizationQml
     URI "QtDataVisualization"
